core/src/main/java/org/apache/accumulo/core/metrics/MetricsUtil.java

```diff
@@ -79,9 +79,14 @@ private static void initializeMetrics(boolean enabled, boolean jvmMetricsEnabled
       List<Tag> tags = new ArrayList<>();
       tags.add(Tag.of("process.name", processName));
 
-      if (address != null) {
-        tags.add(Tag.of("Address", address.toString()));
+      if (address != null && !address.getHost().isEmpty()) {
+        tags.add(Tag.of("host", address.getHost()));
       }
+
+      if (address != null && address.getPort() > 0) {
+        tags.add(Tag.of("port", Integer.toString(address.getPort())));
+      }
+
```

Comment:
Use a nested if statement since both checks rely on the same primary condition. 

```suggestion
      if (address != null) {
        if (!address.getHost().isEmpty()) {
          tags.add(Tag.of("host", address.getHost()));
        }
        if (address.getPort() > 0) {
          tags.add(Tag.of("port", Integer.toString(address.getPort())));
        }
     } 
```

Reply:
addressed in aed77f8e67